elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/schema/SQLTimeDimensionColumn.java

```diff
@@ -50,4 +55,21 @@ public TimeZone getTimeZone() {
     public Set<TimeGrainDefinition> getSupportedGrains() {
         return ((TimeDimensionColumn) wrapped).getSupportedGrains();
     }
+
+    /**
+     * Returns a String that identifies this dimension in a SQL query.
+     * @param requestedGrain The requested time grain.
+     *
+     * @return Something like "table_alias.column_name"
+     */
+    public String getColumnReference(TimeGrain requestedGrain) {
+        Preconditions.checkArgument(getSupportedGrains().stream()
```

Comment:
Why don't we do `findFirst().orElseThrow()` here?

Reply:
👍